src/components/SplashScreenTemplate/SplashScreenTemplate.module.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../sharedStyles.scss";
+@import "../../colours.scss";
+
+.containerLink {
+  background-size: cover;
+  background-position: center;
+  background-blend-mode: overlay;
+  height: 100vh; // Ensure it takes full viewport height
+}
+
+.hasBackground {
+  background-image: url(""); // background added in tsx file given as a prop by user
```

Comment:
??

Reply:
the background url is given by the user later a a prop, this is like a placeholder for it, should i remove it?